lib/inets/doc/src/httpc.xml

```diff
@@ -66,6 +66,7 @@
     <marker id="DATA_TYPES"></marker>
     <p>Type definitions that are used more than once in
       this module:</p>
+    <p><c>autoredirectoptions() = true | false | force</c></p>
```

Comment:
@IngelaAndin How about we do automatic redirect for all codes but keep force just for 303? Will that do?